beacon_node/eth2_libp2p/src/types/mod.rs

```diff
@@ -8,12 +8,14 @@ mod topics;
 use types::{BitVector, EthSpec};
 
 #[allow(type_alias_bounds)]
-pub type EnrBitfield<T: EthSpec> = BitVector<T::SubnetBitfieldLength>;
+pub type EnrAttestationBitfield<T: EthSpec> = BitVector<T::SubnetBitfieldLength>;
+#[allow(type_alias_bounds)]
+pub type EnrSyncCommitteeBitfield<T: EthSpec> = BitVector<T::SyncCommitteeSubnetCount>;
```

Comment:
I did [a bit of digging](https://github.com/rust-lang/lang-team/blob/master/design-meeting-minutes/2020-07-29-wf-checks-and-ty-aliases.md) into this lint, and it seems that it's still up in the air as to what will happen with these bounds.

For now we can avoid the lint by using:

```rust
pub type EnrAttestationBitfield<T> = BitVector<<T as EthSpec>::SubnetBitfieldLength>;
pub type EnrSyncCommitteeBitfield<T> = BitVector<<T as EthSpec>::SyncCommitteeSubnetCount>;
```

Reply:
Sweet! Thanks.